perception_launch/launch/object_recognition/detection/detection.launch.xml

```diff
@@ -4,7 +4,7 @@
 
   <!-- lidar param -->
   <arg name="input/pointcloud"/>
-  <arg name="lidar_detection_model" default="centerpoint" description="options: `centerpoint`, `apollo`"/>
+  <arg name="lidar_detection_model" default="centerpoint" description="options: `centerpoint`, `apollo`, `pointpainting`"/>
```

Comment:
Cloud you change other files s.t. perception.launch.xml in the same way?

Reply:
https://github.com/tier4/autoware_launch/pull/426/commits/3bf7e891e0fe0060395bc80e82b918aa291244a9